R/decision_tree.R

```diff
@@ -0,0 +1,18 @@
+#' @export
+fit_xy.decision_tree <- function(object,
+                                 x,
+                                 y,
+                                 case_weights = NULL,
+                                 control = parsnip::control_parsnip(),
+                                 ...) {
+
+  if (object$engine == "rpart") {
+    # prodlim::EventHistory.frame() expects a call to `Surv()` (or `Hist()`) on
+    # the left-hand side of the formula
```

Comment:
I've opened an issue for this and offered to do a PR so hopefully this whole function is just temporary